webservices/args.py

```diff
@@ -226,6 +226,15 @@ def make_seek_args(field=fields.Int, description=None):
     'beginning_image_number': fields.List(fields.Str, description=docs.BEGINNING_IMAGE_NUMBER),
     'report_type': fields.List(fields.Str, description='Report type; prefix with "-" to exclude'),
     'is_amended': fields.Bool(description='Report has been amended'),
+    'committee_ids': fields.List(fields.Str, description=docs.COMMITTEE_ID),
+    'min_disbursements_amount': Currency(description='Filter for all amounts greater than a value.'),
+    'max_disbursements_amount': Currency(description='Filter for all amounts less than a value.'),
+    'min_receipts_amount': Currency(description='Filter for all amounts greater than a value.'),
+    'max_receipts_amount': Currency(description='Filter for all amounts less than a value.'),
+    'min_cash_on_hand_amount': Currency(description='Filter for all amounts greater than a value.'),
+    'max_cash_on_hand_amount': Currency(description='Filter for all amounts less than a value.'),
+    'type': fields.List(fields.Str, description=docs.COMMITTEE_TYPE),
+    'auth_committee_candidate_id': fields.List(fields.Str, description=docs.COMMITTEE_ID),
```

Comment:
For your second comment, I can name it `candidate_id`, I don't think there is any conflicts there.  I asked @noahmanger if we was ok with the naming, and I'm certainly open to renaming it, the reason I used that name is to make it somewhat explicit that there is a relation entailed in that parameter.  

I'll change the docs to point to `CANDIDATE_ID`.  Thanks for the catch.